namespaces/live-1.cloud-platform.service.justice.gov.uk/emile-sample-app-dev/resources/ecr.tf

```diff
@@ -14,14 +14,14 @@ provider "aws" {
  */
 module "emile_ecr_credentials" {
   source    = "github.com/ministryofjustice/cloud-platform-terraform-ecr-credentials?ref=3.4"
-  repo_name = "emile-sample-app"
-  team_name = "formbuilder-dev"
+  repo_name = "${var.repo_name}"
+  team_name = "${var.team_name}"
```

Comment:
You need a "providers" block in here, as per this example:

https://github.com/ministryofjustice/cloud-platform-terraform-ecr-credentials/blob/master/examples/ecr.tf